editor/plugins/navigation_polygon_editor_plugin.cpp

```diff
@@ -198,14 +200,53 @@ void NavigationPolygonEditor::_bake_pressed() {
 	button_bake->set_pressed(false);
 
 	ERR_FAIL_NULL(node);
-	Ref<NavigationPolygon> navigation_polygon = node->get_navigation_polygon();
-	if (navigation_polygon.is_null()) {
+	Ref<NavigationPolygon> navigation_mesh = node->get_navigation_polygon();
+	if (navigation_mesh.is_null()) {
```

Comment:
does this rename actually make sense?

Reply:
Yes I do it all the time in other PRs to have as much code similarity between 2D and 3D as possible.

There will be a point where NavigationPolygon as a resource name will disappear because frankly, that name sucks soo much. It is a 2D navigation mesh, not a single polygon, and it constantly confuses users in documentation and discussions, because the "real" polygons are the polygons inside a navigation mesh.